docs/griptape-framework/drivers/src/local_ruleset_driver.py

```diff
@@ -16,9 +16,9 @@
 {
     "rules": [
         {
-            "value": "Always talk like a pirate.",
-        },
-    ],
+            "value": "Always talk like a pirate."
+        }
+    ]
```

Comment:
Can we just do `f.write(Ruleset(...).to_json())`?

Reply:
we can but the point is that the schema doesnt have to be a serialized ruleset

Reply:
Ok that's fine (though we should probably make them so). Can you at least just do `f.write(json.dumps({...whatever schema...}))`